src/components/editPostModal/index.js

```diff
@@ -1,25 +1,49 @@
-import { useState } from 'react'
-import useModal from '../../hooks/useModal'
-import './style.css'
-import Button from '../button'
+import { useState } from "react";
+import useModal from "../../hooks/useModal";
+import "./style.css";
+import Button from "../button";
 
 const EditPostModal = () => {
-  const { closeModal } = useModal()
-  const [message, setMessage] = useState(null)
-  const [text, setText] = useState('')
+  const { closeModal } = useModal();
+  const [message, setMessage] = useState("");
+  const [text, setText] = useState("");
+  const [isEditing, setIsEditing] = useState(false);
+  const [isDeleting, setIsDeleting] = useState(false);
 
   const onChange = (e) => {
-    setText(e.target.value)
-  }
+    setText(e.target.value);
+  };
 
   const onSubmit = () => {
-    setMessage('Submit button was clicked! Closing modal in 2 seconds...')
+    setMessage("Post updated! Closing modal in 2 seconds...");
+    setTimeout(() => {
+      setMessage(null);
+      closeModal();
+    }, 2000);
+  };
+
+  const handleEditClick = () => {
+    setIsEditing(true);
+    setIsDeleting(false);
+  };
+
+  const handleDeleteClick = () => {
+    setIsDeleting(true);
+    setIsEditing(false); 
+  };
 
+  const handleConfirmDelete = () => {
```

Comment:
`handleConfirmDelete` and `onSubmit` are basically identical except for the message string. Can we combine them and take string as an input (possible the timeout time too)